test/OpenTelemetry.Instrumentation.AspNetCore.Tests/BasicTests.cs

```diff
@@ -72,7 +74,10 @@ void ConfigureTestServices(IServiceCollection services)
             // Arrange
             using (var client = this.factory
                 .WithWebHostBuilder(builder =>
-                    builder.ConfigureTestServices(ConfigureTestServices))
+                {
+                    builder.ConfigureTestServices(ConfigureTestServices);
+                    builder.ConfigureLogging(loggingBuilder => loggingBuilder.ClearProviders());
```

Comment:
The instrumentation should work even if logging is enabled, right? If that's the case, maybe have some tests where logging is enabled to test that it works then as well.

Reply:
Yes, added logging case of one of the test. It will confirm that the activity is created even with logging enabled. We probably don't need to add that for each test case. Let me know what you think.